test/unit/test-batched-json.js

```diff
@@ -44,7 +44,7 @@ describe('batchFetchJsonFor', () => {
   beforeEach(() => {
     urlReplacements = {
       expandUrlAsync: window.sandbox.stub(),
-      collectUnwhitelistedVarsSync: window.sandbox.stub(),
+      collectUnallowlistedVarsSync: window.sandbox.stub(),
```

Comment:
Doesn't have to be addressed now but it's worth noting there is some inconsistency throughout with regards to "unallowlisted" versus "nonallowlisted"